packages/openneuro-server/graphql/resolvers/snapshots.js

```diff
@@ -21,6 +22,31 @@ export const snapshot = (obj, { datasetId, tag }, context) => {
   }))
 }
 
+export const participantCount = async () => {
+  const aggregateResult = await SnapshotModel.aggregate([
+    { $sort: { created: -1 } },
+    {
+      $group: {
+        _id: { datasetId: '$datasetId' },
+        hexsha: { $last: '$hexsha' },
+      },
+    },
+    {
+      $lookup: {
+        from: 'summaries',
+        localField: 'hexsha',
+        foreignField: 'id',
+        as: 'summary',
+      },
+    },
+    { $project: { subjects: { $size: '$summary.subjects' } } },
+    { $group: { _id: null, participantCount: { $sum: '$subjects' } } },
+  ]).exec()
+  return Array.isArray(aggregateResult)
+    ? aggregateResult[0].participantCount
+    : aggregateResult
```

Comment:
This seems like it would be unnecessary, when would the non-array result be useful? If it's the null case, I'd make it more explicit and return the null.